Makefile

```diff
@@ -167,9 +190,21 @@ release-upload-cluster-controller: release-cluster-controller upload-artifacts
 upload-artifacts:
 	controllers/build/upload_artifacts.sh $(TAR_PATH) $(ARTIFACTS_BUCKET) $(PROJECT_PATH) $(CODEBUILD_BUILD_NUMBER) $(CODEBUILD_RESOLVED_SOURCE_VERSION)
 
-.PHONY: cluster-controller-binaries
-cluster-controller-binaries:
-	controllers/build/create_binaries.sh $(GOLANG_VERSION) $(BINARY_NAME) $(IMAGE_REPO) $(IMAGE_TAG)
+PLATFORMS = linux-amd64 linux-arm64
+create-cluster-controller-binaries: $(foreach platform,$(PLATFORMS),create-cluster-controller-binary-$(platform))
+
+create-cluster-controller-binary-%:
+	CGO_ENABLED=0 GOOS=$(firstword $(subst -, ,$*)) GOARCH=$(lastword $(subst -, ,$*)) $(MAKE) build-cluster-controller-binaries
+	mkdir -p $(OUTPUT_BIN_DIR)/$(BINARY_NAME)/$*/
+	mv bin/manager $(OUTPUT_BIN_DIR)/$(BINARY_NAME)/$*/
+
+.PHONY: cluster-controller-binaries 
+cluster-controller-binaries: $(OUTPUT_BIN_DIR)
+	mkdir -p $(OUTPUT_BIN_DIR)/$(BINARY_NAME)
+	source ./scripts/common.sh && build::common::use_go_version $(GOLANG_VERSION) && go mod vendor
+	$(MAKE) create-cluster-controller-binaries
```

Comment:
Its minor, but can you put the use_go_version stuff around the go build calls as well just to make sure we are using the version we expect?  

We havent been using it, but I believe you can tell make to use a single shell for all the commands, if you'd rather turn that on instead of sourcing + calling use_go_version more than once?